dbt_platform_helper/utils/files.py

```diff
@@ -1,9 +1,41 @@
+from copy import deepcopy
 from os import makedirs
 from pathlib import Path
 
 import click
 import yaml
 
+PLATFORM_CONFIG_FILE = "platform-config.yml"
+
+CONFIG_FILE_MESSAGES = {
+    "storage.yml": " under the key 'extensions'",
+    "extensions.yml": " under the key 'extensions'",
+    "pipelines.yml": ", change the key 'codebases' to 'codebase_pipelines'",
+}
+
+
+def config_file_check():
+    platform_config_exists = Path(PLATFORM_CONFIG_FILE).exists()
+    errors = []
+
+    for file in CONFIG_FILE_MESSAGES.keys():
+        if Path(file).exists():
+            if platform_config_exists:
+                message = f"`{file}` has been superseded by `{PLATFORM_CONFIG_FILE}` and should be deleted."
+            else:
+                message = f"`{file}` is no longer supported. Please move its contents into a file named `{PLATFORM_CONFIG_FILE}`{CONFIG_FILE_MESSAGES[file]} and delete `{file}`."
+            errors.append(message)
+
+    if not errors and not platform_config_exists:
+        errors.append(
+            f"`{PLATFORM_CONFIG_FILE}` is missing. Please check it exists and you are in the root directory of your deployment project."
+        )
+
+    if errors:
+        click.secho("\n".join(errors), bg="red")
+        if not platform_config_exists:
+            exit(1)
```

Comment:
Why would we show errors, but not exit?

Reply:
Because a project may have both platform-config.yml and pipelines.yml. We want to warn them that pipelines.yml is no longer supported but continue with the command as we *do* have a platform-config.yml" 

See Line 228 in: https://github.com/uktrade/platform-tools/pull/461/files#diff-378b3506d661914fd3e66aa8b0e15a2fa7c7e6f9c75cb9c229b98ea6d58b0bb5

Reply:
Using only the config from `platform-config.yml`?

Line 228 in which file?

Reply:
test_files.py. Thought that link would link to the actual file, but apparently not

Reply:
Are you saying it could grab some config from `platform-config.yml` , but use the stuff in `pipelines.yml`?

If yes, I think once a codebase has a `platform-config.yml`, it should just error if any of the lagacy files are still poresent.